pkg/doc/verifiable/credential_jws.go

```diff
@@ -0,0 +1,129 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package verifiable
+
+import (
+	"fmt"
+
+	"github.com/square/go-jose/v3"
+	"github.com/square/go-jose/v3/jwt"
+)
+
+// JWSAlgorithm defines JWT signature algorithms of Verifiable Credential
+type JWSAlgorithm int
+
+const (
+	// RS256 JWT Algorithm
+	RS256 JWSAlgorithm = iota
+
+	// EdDSA JWT Algorithm
+	EdDSA
+
+	// TODO support ES256K (https://github.com/square/go-jose/issues/263)
+)
+
+// Jose converts JWSAlgorithm to JOSE one.
+func (ja JWSAlgorithm) Jose() jose.SignatureAlgorithm {
+	switch ja {
+	case RS256:
+		return jose.RS256
+	case EdDSA:
+		return jose.EdDSA
+	default:
+		logger.Errorf("Unsupported algorithm: %v, fallback to RS256", ja)
```

Comment:
this is a Warnf, as we're not returning an error

also, do we want to log the algorithm (could be a source of security risk)